packages/11ty/content/pdf-copyright.md

```diff
@@ -0,0 +1,9 @@
+---
+class: backmatter
+layout: base.11ty.js
+order: 4
+outputs:
+  - pdf
+title: Copyright
+---
+
```

Comment:
@geealbers Two questions for you:
1) Are the title, half-title, and copyright pages also included in EPUB?
2) What example content should the starter copyright page include here?

Reply:
@hbalenda to #1, yes, those should appear in the EPUB as well. 

For #2, To start with the most basic, I would include the copyright info (as it appears in the online sidebar menu though I'd remove the publisher logo[s]) and then also include the ISBN if one is provided:

```
{% copyright %}

{% if publication.identifier.isbn %}
ISBN: {{ publication.identifier.isbn }}
{% endif %}
```